### PR TITLE
Pagination proposal - namespaces

### PR DESCRIPTION
**Pull request by dgarcia360**

### CODE REVIEW

**Comment by Vektrat** [comment]
we will add the `aliasType` filter and see how it performs since it is not indexed

**Comment by Vektrat** [comment]
minor: i'd rename the following to singular second arg:
```
/namespaces/account/names
/namespaces/mosaic/names
```

⚠️  also on those endpoints, and not so minor, they don't have any params defined to input the list of accounts/mosaics

also just like in the mosaics proposal i don't like much how this endpoint is designed (requesting a bunch of resolutions at once and returning a mixed bag of results), but i won't get into the same spiral here plus in this case it is probably more generic (it is probably more common to request various name resolutions than various mosaic listings per different accounts)

**Comment by Vektrat** [comment]
same as in the mosaics proposal

`/account/namespaces` is being dropped in favour of nothing, will take whatever outcome from that proposal and apply it here (if in the end we add a new endpoint it will need adding)

**Comment by dgarcia360** [comment]
> e the following to singular second arg:

Updated

> /account/namespaces is being dropped in favour of nothing, will take whatever outcome from that proposal and apply it here (if in the end we add a new endpoint it will need adding)

In favor of ``ownerAddress`` query param.

**Comment by Vektrat** [comment]
> > /account/namespaces is being dropped in favour of nothing, will take whatever outcome from that proposal and apply it here (if in the end we add a new endpoint it will need adding)
> 
> In favor of `ownerAddress` query param.

sorry i meant the `POST` version of this with multiple addresses, after all, the mosaics version was made after this one ^^

**Comment by dgarcia360** [comment]
Updated branch to match latest master

**Comment by dgarcia360** [comment]
@Vektrat well seen, added a new request body that only accepts addresses.

File: spec/core/transaction/routes/getPartialTransactions.yml [hunk 1]
```diff
@@ -18,4 +18,4 @@ responses:
   "400":
     $ref: "../../../responses/InvalidContent.yml"
   "409":
-    $ref: "../../../responses/InvalidArgument.yml"
+    $ref: "../../../responses/InvalidArgument.yml"
```
Comment (rg911) [inline spec/core/transaction/routes/getPartialTransactions.yml:21]: new line

File: spec/parameters/_index.yml [hunk 2]
```diff
@@ -34,6 +36,8 @@ heightQuery:
   $ref: './query/height.yml'
 idQuery:
   $ref: './query/id.yml'
+level0Query:
+  $ref: './query/level0.yml'
```
Comment (rg911) [inline spec/parameters/_index.yml:40]: would it be better to use words instead? e.g. `rootLevel` / `levelZero` etc..

Comment (Vektrat) [reply spec/parameters/_index.yml:40]: this is how it is in the database

**Pull request merged**